src/cluster/src/install.rs

```diff
@@ -58,14 +59,20 @@ pub struct ClusterInstallerBuilder {
     chart_name: String,
     /// A specific version of the Fluvio helm chart to install
     chart_version: String,
-    /// The location to find the helm chart
+    /// The location to find the fluvio app chart
     chart_location: ChartLocation,
+    /// The location to find the fluvio sys chart
+    chart_location_sys: ChartLocation,
     /// The name of the SPU group to create
     group_name: String,
     /// The name of the Fluvio cloud
     cloud: String,
     /// Whether to save an update to the Fluvio profile
     save_profile: bool,
+    /// Whether to install fluvio-sys with fluvio-app
+    install_sys: bool,
+    /// Whether to update the `kubectl` context
+    update_context: bool,
```

Comment:
can we group both `install_sys` and `update_context` into separate option?  likely, there would be more option so we can be forward compatible

Reply:
I'm not sure what you mean by grouping them into a separate option. They are two different configurations.
- `install_sys` causes the installer to check if `fluvio-sys` is installed, and to install it automatically if not
- `update_context` causes the installer to update the `kubectl` context and, if necessary, the /etc/hosts entry